File: src/ExamenApi.Domain/Entities/Contacto.cs
namespace ExamenApi.Domain.Entities
{
    public class Contacto
    {
        public int Id { get; set; }
        public string Nombre { get; set; } = string.Empty;
        public string Telefono { get; set; } = string.Empty;
        public string Direccion { get; set; } = string.Empty;
        public double Latitud { get; set; }
        public double Longitud { get; set; }
        public string? FirmaBase64 { get; set; }
        public string? ImagenBase64 { get; set; }
        public DateTime FechaCreacion { get; set; } = DateTime.Now;

        public int UsuarioId { get; set; }
        public Usuario? Usuario { get; set; } 
    }
}

File: CloudContacts/app/src/main/java/com/uth/cloudcontacts/domain/model/Contacto.kt
package com.uth.cloudcontacts.domain.model

data class Contacto(
    val id: Int = 0,
    val nombre: String,
    val telefono: String,
    val latitud: Double,
    val longitud: Double,
    val imagenBase64: String?,
    val usuarioId: Int
)
